molecule/testinfra/staging/mon/iptables-mon-prod.j2

```diff
@@ -19,12 +19,12 @@
 -A INPUT -i lo -m comment --comment "Allow lo to lo traffic all protocols" -j ACCEPT
```

Comment:
Same point as above for the prod iptables template.

Reply:
:+1: